content/docs/rfcs/11/README.md

```diff
@@ -70,65 +88,88 @@ message RPC {
 
-These are messages sent to directly connected peers. They SHOULD NOT be
-gossiped. See section below on how the fields work.
+> **_NOTE:_**
+The [`TopicDescriptor`](https://github.com/libp2p/specs/blob/master/pubsub/README.md#the-topic-descriptor) is not currently used by `WakuRelay`.
 
-## Message
+## Message fields
 
-The `from` field MAY indicate which peer is publishing the message.
+The `Message` protobuf defines the format in which content is relayed between peers.
+`WakuRelay` specifies the following usage requirements for each field:
 
-The `data` field MUST be filled out with a `WakuMessage`. See the [Waku Message](waku-message.md) spec for more details.
+- The `from` field MUST NOT be used, following the [`StrictNoSign` signature policy](#Signature-Policy).
 
-The `seqno` field MAY be used to provide a linearly increasing number. See PubSub spec for more details.
+- The `data` field MUST be filled out with a `WakuMessage`.
+See the [Waku Message](waku-message.md) spec for more details.
 
-The `topicIDs` field MUST contain the topics that a message is being published on.
+- The `seqno` field MUST NOT be used, following the [`StrictNoSign` signature policy](#Signature-Policy).
 
-The `signature` field MAY contain the signature of the message, thus providing authentication of the message. See PubSub spec for more details.
+- The `topicIDs` field MUST contain the topics that a message is being published on.
 
-The `key` field MAY be present and relates to signing. See PubSub spec for more details.
+- The `signature` field MUST NOT be used, following the [`StrictNoSign` signature policy](#Signature-Policy).
 
+- The `key` field MUST NOT be used, following the [`StrictNoSign` signature policy](#Signature-Policy).
 
-## SubOpts
+## SubOpts fields
 
-To do topic subscription management, we MAY send updates using `SubOpts` to our peers. If we do so, then:
+The `SubOpts` protobuf defines the format in which subscription options are relayed between peers.
+A `WakuRelay` node MAY decide to subscribe or unsubscribe from topics by sending updates using `SubOpts`.
+The following usage requirements apply:
 
-The `subscribe` field MUST contain a boolean, where 1 means subscribe and 0 means unsubscribe to a topic.
+- The `subscribe` field MUST contain a boolean, where `true` indicates subscribe and `false` indicates unsubscribe to a topic.
 
-The `topicid` field MUST contain the topic.
+- The `topicid` field MUST contain the topic.
 
 ## Signature Policy
 
-The `StrictNoSign` option MUST be used. 
+The [`StrictNoSign` option](https://github.com/libp2p/specs/blob/master/pubsub/README.md#signature-policy-options) MUST be used, to ensure that messages are built without the `signature`, `key`, `from` and `seqno` fields. Note that this does not merely imply that these fields be empty, but that they MUST be _absent_ from the marshalled message.
```

Comment:
What's the significance of this distinction?

Reply:
Use sembr

Reply:
There's a slight semantic difference between an empty field and an absent field in protobuf (an empty field will still have a populated field tag number when marshalled, which could lead to dodgy interpretations when unmarshalled). The latter is specified by PubSub for `StrictNoSign`, but repeated here as it may be an important implementation requirement for `11/WAKU2-RELAY`.